src/sdk/Transcription/ConversationTranslator.ts

```diff
@@ -194,7 +194,7 @@ export class ConversationTranslator implements IConversationTranslator, IDisposa
         try {
 
             // stop the speech websocket
-            this.cancelSpeech();
+            this.cancelSpeech().catch();
```

Comment:
>.catch(); [](start = 31, length = 9)

Review

Reply:
Isn't there supposed to be a function handler here for the error cases?

---
In reply to: [449317700](https://github.com/microsoft/cognitive-services-speech-sdk-js/pull/198#discussion_r449317700) [](ancestors = 449317700)

Reply:
Yes. I reviewed all the empty catch() statements and added something. They were initially there to help w/ the linter, and to serve as easily findable review targets.

---
In reply to: [457662561](https://github.com/microsoft/cognitive-services-speech-sdk-js/pull/198#discussion_r457662561) [](ancestors = 457662561,449317700)